src/store/wallet.js

```diff
@@ -46,6 +51,32 @@ function _useWallet(initialWallet = Nothing(), initialMnemonic = Nothing()) {
   const [networkSeed, setNetworkSeed] = useState(Nothing());
   const [networkRevision, setNetworkRevision] = useState(Nothing());
 
+  const [authToken, setAuthToken] = useState(Nothing());
```

Comment:
I thought nobody liked the `maybe` objects and we were gonna not use them for new code? They're relatively common across the rest of `wallet.js` still though, so probably better for local consistency, but it's exposed, too...

Reply:
Yeah maybe kind of sucks but it's pervasive across the entire codebase. We ought to re-evaluate how/where/when we store data in the general but I don't think this PR is the right place to have that conversation